table/render_html_test.go

```diff
@@ -517,3 +517,29 @@ func TestTable_RenderHTML_Sorted(t *testing.T) {
   </tfoot>
 </table>`)
 }
+
+func TestTable_RenderHTML_RowAutoMerge(t *testing.T) {
+	rcAutoMerge := RowConfig{AutoMerge: true}
+	tw := NewWriter()
+	tw.AppendHeader(Row{"A", "B"})
+	tw.AppendRow(Row{"Y", "Y"}, rcAutoMerge)
+	tw.AppendRow(Row{"Y", "N"}, rcAutoMerge)
+	compareOutput(t, tw.RenderHTML(), `
+<table class="go-pretty-table">
+  <thead>
+  <tr>
+    <th>A</th>
+    <th>B</th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td align="center" colspan=2>Y</td>
+  </tr>
+  <tr>
+    <td>Y</td>
+    <td>N</td>
+  </tr>
+  </tbody>
+</table>`)
+}
```

Comment:
Can you add a test with a more complex use-case... with a couple of columns without any merge in the same row where a couple of columns are merged? Check the automerge test file for ideas?

Reply:
Sure. I've forced pushed with an extra test that hopefully covers enough alternative merge/non-merge options.